src/lib/aws-http.client.ts

```diff
@@ -10,9 +10,13 @@ export declare type HttpObserve = 'body' | 'events' | 'response';
 
 export interface RequestOptions {
 	body?: any;
-	headers?: HttpHeaders;
+	headers?: HttpHeaders | {
```

Comment:
Are you sure about this change? Any link to documentation indicating that this should work?

Reply:
https://angular.io/api/common/http/HttpClient#request

Reply:
:+1: